### PR TITLE
Enable barebones Xwalk compatibility

### PR DESCRIPTION
**Pull request by dcousens**
What shows:
* Camera overlay

What doesn't show: 
* WebView overlay

This isn't ready to merge, as `setAlpha(0.0f)` completely destroys this packages original premise... but **it is a start for maybe someone else to take over and or use as an "almost there" solution.**

For me,  it gives enough headroom (combined with an informational "toast") to inform them they can "tap-to-exit",  and allow me to promptly migrate away from Crosswalk...

The WebView HTML overlay is actually there, and can be interacted with... it just isn't showing.

I don't know why - @bitjson if you can see why,  please fix :dancer:

### CODE REVIEW

File: src/android/QRScanner.java [hunk 1]
```diff
@@ -619,7 +642,18 @@ private void show(final CallbackContext callbackContext) {
...
+                try {
+                    Method getView = webView.getClass().getMethod("getView");
+                    vv = (View) getView.invoke(webView);
+                } catch (Exception e) {
+                    e.printStackTrace();
+                    return;
+                }
+
+                vv.setBackgroundColor(Color.TRANSPARENT);
+                vv.setAlpha(0.0f);
```
Comment (dcousens) [inline src/android/QRScanner.java]: `setBackgroundColor` here does ... nothing.

File: src/android/QRScanner.java [hunk 2]
```diff
@@ -425,7 +429,17 @@ private void makeOpaque() {
...
             @Override
             public void run() {
-                webView.getView().setBackgroundColor(Color.WHITE);
+                View vv;
+                try {
+                    Method getView = webView.getClass().getMethod("getView");
+                    vv = (View) getView.invoke(webView);
+                } catch (Exception e) {
+                    e.printStackTrace();
+                    return;
```
Comment (dcousens) [inline src/android/QRScanner.java]: Error should probably go through the callback...

**Comment by dcousens** [comment]
ping @gildebrand @dPary

File: src/android/QRScanner.java (hunk 1 above)
Comment (dcousens) [inline src/android/QRScanner.java]: @nolanlawson any tips?

**Comment by dcousens** [comment]
8a43420 has been added purely for using this branch in a test deployment,  it can be removed if this PR is progressed

**Comment by raxim1** [comment]
I'm not familiar with the cordova development. To be honest it stinks and sucks,
I was able to test on android latest emulator, and made the following changes:

I have removed this setBackground stuff. You don't need to tweak with setBackground stuff.
Anyway every suggested idea was not working.

It would be beneficial also, if there would be some standalone example which works in android studio.
To import this project to android studio is a nightmare. Sources here and there.

It's a partial solution, because i don't see any buttons on the screen. Just a full camera screen.

```
private void show(final CallbackContext callbackContext) {
        this.cordova.getActivity().runOnUiThread(new Runnable() {
            @Override
            public void run() {
                mBarcodeView.bringToFront();
                showing = true;
                getStatus(callbackContext);
            }
        });
    }
    private void hide(final CallbackContext callbackContext) {
        webView.getView().bringToFront();
        getStatus(callbackContext);
    }

```

**Pull request closed without merging**